lb.tf

```diff
@@ -7,6 +7,12 @@ resource "aws_lb" "app_lb" {
   tags {
     Name = "Application LB ${var.app_name}"
   }
+
+  # sometimes updating an lb will cause a destroy then create operation, rather than update in place, we want to prevent
+  # this so we don't cause an outage.
+  lifecycle {
+    prevent_destroy = true
+  }
```

Comment:
Please turn this into a variable with a default value instead of a hard coded item.